static/docs/command-reference/remote/modify.md

```diff
@@ -128,6 +128,42 @@ these settings, you could use the following options:
   $ dvc remote modify myremote acl bucket-owner-full-control
   ```
 
+- `grant_read`\* - grants `READ` permissions at object level access control list
+  for specific grantees\*\*. Grantee can read object and its metadata.
+
+  ```dvc
+  $ dvc remote modify myremote grant_read id=aws-canonical-user-id,id=another-aws-canonical-user-id
+  ```
+
+- `grant_read_acp`\* - grants `READ_ACP` permissions at object level access
+  control list for specific grantees\*\*. Grantee can read the object's ACP.
+
+  ```dvc
+  $ dvc remote modify myremote grant_read_acp id=aws-canonical-user-id,id=another-aws-canonical-user-id
+  ```
+
+- `grant_write_acp`\* - grants `WRITE_ACP` permissions at object level access
+  control list for specific grantees\*\*. Grantee can modify the object's ACP.
+
+  ```dvc
+  $ dvc remote modify myremote grant_write_acp id=aws-canonical-user-id,id=another-aws-canonical-user-id
+  ```
+
+- `grant_full_control`\* - grants `FULL_CONTROL` permissions at object level
+  access control list for specific grantees\*\*. Equivalent of grant_read +
+  grant_read_acp + grant_write_acp
+
+  ```dvc
+  $ dvc remote modify myremote grant_full_control id=aws-canonical-user-id,id=another-aws-canonical-user-id
+  ```
+
+  > \* - `grant_read`, `grant_read_acp`, `grant_write_acp` and
+  > `grant_full_control` options are mutually exclusive with `acl` option.
+  >
+  > \*\* - default ACL grantees are overwritten. Grantees are AWS accounts
+  > identifiable by `id` (AWS Canonical User ID), `emailAddress` or `uri`
+  > (predefined group).
+
```

Comment:
@rbvesb , what do you think about adding the following reference link: [ACL overview](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#permissions)

Reply:
Good idea @mroutis , thanks!
<img width="708" alt="dvc-remote-s3-acls-references" src="https://user-images.githubusercontent.com/42961356/70706401-7e917200-1cde-11ea-9615-a7fc6fcaf6a4.png">

Reply:
Nice! My impression is this thing is a little long already though, let me see if I can come up with a good suggestion in a separate review...